### ISSUE TITLE
Appearance of app windows

### ISSUE THREAD

**Issue by Valdnet** [SmartPack/PackageManager#55]
I would propose to improve the appearance:

1. The initial application window that appears at the first launch.
2. Application window about granting permissions.
3. Application window about canceling installation of the installation file.
4. Application window that appears after the installation of the installation file.

They should be matched to the appearance of the entire application. Can change buttons with text string. Leave only text strings with matching colors.

Overall, it doesn't quite match the overall look.

Some screenshots:

![obraz](https://user-images.githubusercontent.com/47037905/133321041-fd3cb5ff-0053-4a8e-b3fe-975081d30c87.png)
---
![obraz](https://user-images.githubusercontent.com/47037905/133320795-9d5276cf-30d2-45b3-ac0e-b47ca918d00d.png)
---
![obraz](https://user-images.githubusercontent.com/47037905/133320734-a7c01e25-c3fe-422f-b902-0129ec1d4b8d.png)
---
![obraz](https://user-images.githubusercontent.com/47037905/133401160-ae06ff18-85e7-433c-8898-e810f79bf21e.png)

**Comment by sunilpaulmathew** [comment]
You're perfectly right. But, I'll postponed this (post-vacation). :)

**Comment by sunilpaulmathew** [comment]
@Valdnet Okay, I did this too. Is something similar missing?
![Screenshot_1631653256](https://user-images.githubusercontent.com/15515387/133333378-dad4add1-567d-4664-b409-9c82ed8a35d9.png)

**Comment by Valdnet** [comment]
@sunilpaulmathew I would give up on buttons. I would leave the text string alone.
Is the font bold? If so, I would try to set the font to normal. It's too conspicuous 😉.

> Okay, I did this too. Is something similar missing?

Changes should be made for all messages that appear in the application from the first launch of the application.

**Comment by sunilpaulmathew** [comment]
> s should be made for all messages that appear in the application from the first launch of the

Only time (or more testing) can tell if some missing or not. Probably, something in the root  related tasks which I will check soon.

**Comment by Valdnet** [comment]
Also remember about those windows mentioned above in issue 😉.

**Comment by sunilpaulmathew** [comment]
> Also remember about those windows mentioned above in issue 😉.

Of course, but

> 3. Application window about canceling installation of the installation file

What;' the problem here?

Btw, found one more.
![Screenshot_20210915-104735](https://user-images.githubusercontent.com/15515387/133402434-d5c31091-9b0e-4188-8536-aa19cb3ab852.jpg)

**Comment by Valdnet** [comment]
> > 3. Application window about canceling installation of the installation file
> 
> What;' the problem here?

It's about this window:

![obraz](https://user-images.githubusercontent.com/47037905/133320734-a7c01e25-c3fe-422f-b902-0129ec1d4b8d.png)

**Comment by sunilpaulmathew** [comment]
> It's about this window:

I understand, but isn't it consistent with other screens (may be I'm blind).

**Comment by Valdnet** [comment]
The point is, all the windows I mentioned look a little different than the rest of the windows in the app. I mean correct the appearance of text strings and the removal of the button. Its description is enough. It doesn't match the modern look.

**Comment by sunilpaulmathew** [comment]
> I mean correct the appearance of text strings and the removal of the button.

Yes, the text need to be adjusted (remove bold characters, adjust size, etc.) throughout the app. By the way, are you suggesting to remove blue card around the buttons on the bottom? Either way, I'm afraid it won't finish before the next update (planning tonight or tomorrow morning).

**Comment by Valdnet** [comment]
> Either way, I'm afraid it won't finish before the next update (planning tonight or tomorrow morning).

Yes. I suggest removing the button. Just the text string alone will suffice.
Something like this:

![obraz](https://user-images.githubusercontent.com/47037905/133422895-f9be7db2-1a21-4d58-bc67-93a30aa83916.png)

Now button developers no longer use in applications.

> Either way, I'm afraid it won't finish before the next update (planning tonight or tomorrow morning).

We will wait 😉.

**Comment by sunilpaulmathew** [comment]
> Now button developers no longer use in applications.

In fact, its is not a button. It is a MaterialCardView wrapped around the text. I also do not like using buttons :)

**Comment by sunilpaulmathew** [comment]
@Aga-C @Valdnet 
Thanks a lot to both of you. I finally managed to submit a new version (v5.3) to Google Play and F-Droid. Because of your help, v5.3 should be much better (in UI & functionalities) than before. Since the F-Droid release usually takes days, [here is an APK](https://www.dropbox.com/s/rvi05lbi7yeya2i/com.smartpack.packagemanager.apk?dl=0) built exclusively for you people 😉 Also, I added a separate section on the [credit page](https://github.com/SmartPack/PackageManager/blob/master/Credits.md#ui-improvements) only to acknowledge both of you (I know its not enough, but at least something).
So, thanks a lot once again. Now I can relax a bit.
